services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -47,13 +60,90 @@ const DesignKits = ({ libraryData, navData }) => {
     )
   }
 
+  let groupedDesignKits
+  if (libraryData.content.designKits) {
+    groupedDesignKits = groupBy(Object.values(libraryData.content.designKits), (kit) => kit.tool)
+  }
+
+  let filteredDesignTools
+  if (libraryData.content.designKits) {
+    filteredDesignTools = Object.keys(designTools).filter((item) =>
+      Object.keys(groupedDesignKits).includes(item)
+    )
+  }
+
+  const NoDesignKits = () => {
+    return (
+      <>
+        <PageDescription>This library contains no compatible design kits.</PageDescription>
+        <P>
+          It looks like there are no design kits compatible with this library. Explore the design
+          kit catalog to find more design kits or contact libray maintainers.
+        </P>
+        <CardGroup>
+          <Column sm={4} md={4} lg={4}>
+            <ResourceCard subTitle="Design kit catalog" href="/design-kits">
+              <RulerAlt size={32} />
+            </ResourceCard>
+          </Column>
+        </CardGroup>
+      </>
+    )
+  }
+
   return (
     <>
       <NextSeo {...seo} />
       <PageHeader bgColor={pageHeader?.bgColor} title="Design kits" pictogram={pageHeader?.icon} />
       <Grid>
         <Column sm={4} md={8} lg={12}>
-          <div className={pageStyles.content}>Coming soon!</div>
+          <div className={pageStyles.content}>
+            {!libraryData.content.designKits && <NoDesignKits />}
+            {libraryData.content.designKits && (
+              <>
+                <PageDescription>
+                  The following design kits are compatible with the {name} library. If you are new
+                  to Carbon, check out the{' '}
+                  <Link href="/designing/get-started" passHref>
+                    <CarbonLink>design guidance</CarbonLink>
+                  </Link>{' '}
+                  to get started. You can also view all design kits in the{' '}
+                  <Link href="/design-kits" passHref>
+                    <CarbonLink>catalog</CarbonLink>
+                  </Link>
+                  .
+                </PageDescription>
+                <AnchorLinks>
+                  {filteredDesignTools.length > 2 &&
+                    filteredDesignTools.map((item, i) => (
+                      <AnchorLink key={i}>{designTools?.[item]?.name}</AnchorLink>
+                    ))}
+                </AnchorLinks>
```

Comment:
is the spacing here correct? maybe the conditional for `.length >2` should be outisde of `<AnchorLinks>` so the extra spacing doesn't render?
<img width="1498" alt="image" src="https://user-images.githubusercontent.com/40550942/189932599-fc817264-ce66-441d-93c6-9d47534826ef.png">

Reply:
Oh yea, good catch!